SAMv1.tex

```diff
@@ -344,28 +353,30 @@ \subsection{The alignment section: mandatory fields}
   \item Sum of lengths of the {\tt M/I/S/=/X} operations shall equal
     the length of {\sf SEQ}.
   \end{itemize}
-\item {\sf RNEXT}: Reference sequence name of the primary alignment of the NEXT read in the
-  template. For the last read, the next read is the first
-  read in the template. If {\tt @SQ} header lines are present, {\sf
-    RNEXT} (if not `*' or `=') must be present in one of the {\tt SQ-SN}
-  tag. This field is set as `*' when the information is unavailable, and
-  set as `=' if {\sf RNEXT} is identical {\sf RNAME}. If not `=' and the
-  next read in the template has one primary mapping (see also bit
-  0x100 in {\sf FLAG}), this field is identical to {\sf RNAME} at the primary line of the
-  next read.  If {\sf
-    RNEXT} is `*', no assumptions can be made on {\sf PNEXT} and bit
+\item {\sf RNEXT}: Reference sequence name of the NEXT read in this
+  template alignment, where NEXT is defined to be the next read in
```

Comment:
So essentially this PR essentially redefines secondary alignments as secondary alignments for the entire template as opposed to the current definition that implicitly (by requiring RNEXT to point to the primary alignment) defines secondary alignments as secondary alignments for the segment.

For read pairs that have one read uniquely mappable and the other read multi-mapping to 100 different location this change in definition will require spec-compliant implementations to write out the uniquely aligned segment 100 times, each with a different RNEXT.

It is a major change that will break many tools as tools can no longer assume that a high mapq means that record is unique. This as simple as calculating read depth of reads with (e.g.) mapq>10 will now require read deduplication. Is this intended?